packages/lexical/src/nodes/LexicalParagraphNode.ts

```diff
@@ -63,11 +64,22 @@ export class ParagraphNode extends ElementNode {
 
   exportDOM(editor: LexicalEditor): DOMExportOutput {
     const {element} = super.exportDOM(editor);
+    const format = this.__format;
+    const dir = this.__dir;
 
     if (element && this.isEmpty()) {
       element.append(document.createElement('br'));
     }
 
+    if (element && format !== 0) {
+      element.style.cssText = 'text-align: ' + ELEMENT_FORMAT_TO_TYPE[format];
+    }
+
+    if (element && dir) {
+      element.setAttribute('dir', dir);
+      element.classList.add('PlaygroundEditorTheme__' + dir);
+    }
+
```

Comment:
The playground class name is user definable and there are some convenience methods on the super class we can use instead:

```suggestion
    if (element) {
      element.style.cssText += 'text-align: ' + this.getFormatType();
    }

    const direction = this.getDirection();

    if (element && direction) {
      element.setAttribute('dir', direction);

      const ltrClassName = editor._config.theme.ltr;
      const rtlClassName = editor._config.theme.rtl;

      if (direction === 'ltr' && ltrClassName) {
        element.classList.add(ltrClassName);
      } else if (direction === 'rtl' && rtlClassName) {
        element.classList.add(rtlClassName);
      }
    }
```

Reply:
Got it!
Is this intentional? As we would be having the format set to 0 when nothing is applied, which results in `text-align: ;` being added
```
'text-align: ' + this.getFormatType();
```

Reply:
Just updated that to be `if (element && this.getFormatType() !== '') {`